Lib/distutils/command/build_ext.py

```diff
@@ -490,7 +490,7 @@ def build_extension(self, ext):
                   "in 'ext_modules' option (extension '%s'), "
                   "'sources' must be present and must be "
                   "a list of source filenames" % ext.name)
-        sources = list(sources)
+        sources = sorted(sources)
```

Comment:
Can you please add a comment to explain why sources are sorted? Just explain that it helps to make the build more reproducible.

Reply:
like this?